changelog.d/+removed-copy-target-checks.changed.md

```diff
@@ -0,0 +1,2 @@
+1. mirrord CLI now does not check target type when the `copy_target` feature is enabled. The check is now done only in the operator.
+2. `mirrord operator setup` not includes permissions to read and change rollouts scale.
```

Comment:
remember to update it in the helm chart repo please.

Reply:
https://github.com/metalbear-co/charts/pull/39